app/views/proposals/index.html.haml

```diff
@@ -1,41 +1,107 @@
 .row
-  .col-sm-12
+  .col-md-12
     .page-header
-      %h1 Your Dashboard
+      %h1 My Proposals
 .row
   .col-md-8.proposals
-    %h2 Proposals
     - if proposals.empty?
       %h3 You don't have any proposals.
     - proposals.each do |event, talks|
-      - if event.open?
-        = link_to 'Submit a proposal', new_proposal_path(slug: event.slug), class: 'btn btn-primary pull-right'
-      %h3.margin-bottom
-        = link_to event.name, event_path(event.slug), class: 'event-title'
-        %span.label.label-info= event.status
-      .widget
-        .widget-header
-          %i.fa.fa-comment
-          %h3 Proposed Talks
-        .widget-content
-          %ul.list-unstyled
-            - talks.each do |proposal|
-              %li.proposal
-                %h4= link_to proposal.title, proposal_path(slug: proposal.event.slug, uuid: proposal)
-                = proposal.public_state(small: true)
-                %p= truncate(proposal.abstract, length: 80)
-                %hr/
+      - event = event.decorate
+
+      .row
+        .col-md-7
+          %h1.inline-block
+            - if event.url?
+              = link_to event.name, event.url, class: 'event-title'
+            - else
+              = link_to event.name, event_path(event.slug), class: 'event-title'
+
+          %span= link_to 'View Guidelines', event_path(event.slug), class: 'guidelines-link'
+
+          %h4.margin-bottom
+            = event.start_date.strftime("%a %b %d") + " - " + event.end_date.strftime("%a %b %d")
```

Comment:
It might be good to add this date format in `config/initializers/time_formats.rb` so we can reuse it elsewhere. There's probably a better name for the format, but using 'day_month_date' as the name:

```
Time::DATE_FORMATS[:day_month_date] = "%a %b %d"
```

```
= event.start_date.to_s(:day_month_date) + " - " + event.end_date.to_s(:day_month_date)
```